internal/exec/terraform_outputs.go

```diff
@@ -296,19 +296,15 @@ func GetTerraformOutput(
 
 	sections, err := ExecuteDescribeComponent(component, stack, true, true, nil)
 	if err != nil {
-		if atmosConfig.Logs.Level == u.LogLevelTrace || atmosConfig.Logs.Level == u.LogLevelDebug {
-			fmt.Printf("\r✗ %s\n", message)
-		}
+		l.Debug(fmt.Sprintf("\r✗ %s\n", message))
```

Comment:
No, this is console output, and shouldn't be considered log output.

Reply:
Done